libvirt.go

```diff
@@ -266,14 +281,19 @@ func (d *Driver) Create() error {
 		return err
 	}
 
+	bridge_name, err := d.getBridgeName()
+	if err != nil {
+		return err
+	}
+
 	config := DomainConfig{
 		DomainName: d.MachineName,
 		Memory:     d.Memory,
 		CPU:        d.CPU,
 		CacheMode:  d.CacheMode,
 		IOMode:     d.IOMode,
 		DiskPath:   d.DiskPath,
-		Network:    d.Network,
+		BridgeName: bridge_name,
```

Comment:
I believe it should be 'bridgeName' (no time to check go coding standards now), but 'bridge_name' is consistent with the rest of the file, so fine with me for now.